README.md

```diff
@@ -72,7 +72,7 @@ existing proposals.
 [1](https://github.com/aws/aws-cdk-rfcs/issues/1)|CDK Watch||[@eladb](https://github.com/eladb)||proposed
 <!--END_TABLE-->
 
-## What does all this mean?!
+## What does all this mean?
```

Comment:
why was this changed? I think the intention of the author was to use "?!"

Reply:
It was removed to follow the guideline of not having punctuations at the end of headers, except for question mark.

>  I think the intention of the author was to use "?!"

And what is this intention? I believe the intent for this question is to be rhetorical, which is conveyed with just the question mark, no? Very similar to the next header - "What is an RFC?" - https://github.com/aws/aws-cdk-rfcs/blob/master/README.md#what-does-all-this-mean